code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1773,6 +1773,7 @@ GLOBAL_LIST_EMPTY(features_by_species)
 
 	var/armor_block = H.run_armor_check(affecting, MELEE, "<span class='notice'>Your armor has protected your [hit_area]!</span>", "<span class='warning'>Your armor has softened a hit to your [hit_area]!</span>",I.armour_penetration)
 	var/Iforce = I.force //to avoid runtimes on the forcesay checks at the bottom. Some items might delete themselves if you drop them. (stunning yourself, ninja swords)
+	var/limb_health = affecting.get_damage() //We need to save this for later to simplify dismemberment
```

Comment:
I don't like that this is copy and pasted, but if this is just how it's done... I'd much rather this be some kind of generic proc on carvon/human which both of these cases call

Reply:
There is a very subtle difference but yeah I was pretty baffled at first as to why it was like this.

Monkeys are the only thing that still uses carbon defense and the PR that moved them to species didn't survive.